gapis/resolve/report.go

```diff
@@ -107,6 +107,13 @@ func (r *ReportResolvable) Resolve(ctx context.Context) (interface{}, error) {
 		}
 	}
 
+	// Start with issues that are not specific to a command, like
```

Comment:
Is this "Start with" because you're going to do related changes in this specific area of code later? Otherwise this comment seems more PR-related.

Reply:
Ok, i guess it ties well with the next one "Gather report items.." ?

Reply:
Yes, it is "start with" in the sense that the piece of code continues with regular commands.